moto/emr/models.py

```diff
@@ -16,6 +16,7 @@
     CamelToUnderscoresWalker,
     EmrSecurityGroupManager,
 )
+from tests import EXAMPLE_AMI_ID
```

Comment:
This won't work.  The `tests` package is not included in the `moto` package that gets uploaded to pypi.  You'll need to define the constant yourself.

Reply:
Removed import from tests.